cdhweb/conftest.py

```diff
@@ -1,3 +1,8 @@
 """Fixtures/utilities that should be globally available for testing."""
 from cdhweb.pages.tests.conftest import site, homepage
-from cdhweb.projects.tests.conftest import projects_link_page
+# FIXME not sure how else to share fixtures that depend on other fixtures
+# between modules - if you import just the top-level fixture (e.g. "events"),
+# it fails to find the fixture dependencies, and so on all the way down. For
+# now this does what we want, although it pollutes the namespace somewhat
```

Comment:
I couldn't find anything in the docs describing a situation where a fixture is shared across modules, rather than just across multiple test files in a module. The closest thing seems to be changing fixture scope, but I still want these to be created on every function call rather than shared :/